e2e/test-config.js

```diff
@@ -0,0 +1,45 @@
+require('dotenv').config();
+const { devices } = require('@playwright/test');
+
+const scenarioConfig = (scenarioName, applicationType) => ({
+    provider: {
+        name: `provider-${scenarioName}`,
+        testMatch: `**/${applicationType}/scenarios/${scenarioName}/provider.spec.js`,
+        use: { ...devices['Desktop Chrome'] },
```

Comment:
If it's easy enough to do (might end up in another ticket instead) it would be good to add other browsers here; or at least be able to run multiple browsers in multiple CI steps

Reply:
separate ticket is advisable, as we might also want to test on mobile devices.

Reply:
Done and yeah mobile devices is what prompted me. https://dsdmoj.atlassian.net/browse/CRM457-2329